docs/src/modules/utils/generatePropTypeDescription.ts

```diff
@@ -47,20 +47,24 @@ function getChained(type: PropTypeDescriptor) {
   return false;
 }
 
-function escapeCell(value: string): string {
+export function escapeCell(value: string): string {
```

Comment:
I added an export here because the "buildApi" file duplicated these functions. 
I deleted the duplicates, exported them from here, and used them there.